fmriprep/workflows/anatomical.py

```diff
@@ -74,43 +74,44 @@ def t1w_preprocessing(settings, name='t1w_preprocessing'):
             testing=settings.get('debug', False),
             template='mni_icbm152_nlin_asym_09c'
         ),
-        name='T1_2_MNI_Registration'
+        name='t1_2_mni'
     )
     # should not be necesssary but does not hurt - make sure the multiproc
     # scheduler knows the resource limits
     t1_2_mni.interface.num_threads = settings['ants_nthreads']
 
     # 6. FreeSurfer reconstruction
     if settings['freesurfer']:
-        surface_recon = surface_reconstruction(settings=settings)
+        surface_recon = surface_reconstruction(name='surface_recon',
```

Comment:
Some thoughts:
1) Let's keep a naming convention for Function functions for another PR
2) We should probably enforce that all Workflow constructors have `_wf` suffix
3) It might be actually beneficial to add `_wf` to the Workflow name - it makes it easier to distinguish Workflow folder from a Node folder in the working directory
4) Also - are functions the most intuitive way of creating subworkflows. Maybe those should just be classes inheriting from Workflow?
4) The scheme you outlined has a lot of rules which will make it hard to follow for newcomers. If we get rid of the sub/super strings things could become easier.